client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import useStyles from './useStyles';
+import { Button, IconButton, Grid, Menu, MenuItem } from '@material-ui/core';
+import Logo from '../../Images/logo.png';
+import { User } from '../../interface/User';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+import { useHistory } from 'react-router-dom';
+
+interface Props {
+  user: User | null | undefined;
+  logout: () => void;
+}
+
+export default function Navbar({ user, logout }: Props) {
+  const classes = useStyles();
+  const history = useHistory();
+
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleLogout = () => {
+    handleClose();
+    logout();
+  };
+
+  const Nav = () => (
+    <>
+      <Button color="inherit" className={`${classes.btn} ${classes.sitterBtn}`} variant="text">
+        become a sitter
+      </Button>
+      <Button
+        color="inherit"
+        className={`${classes.btn} ${classes.loginBtn}`}
+        variant="outlined"
+        onClick={() => history.push('/login')}
+      >
+        Login
+      </Button>
+      <Button
+        color="inherit"
+        className={`${classes.btn} ${classes.signupbtn}`}
+        variant="contained"
+        onClick={() => history.push('/signup')}
+      >
+        signup
+      </Button>
```

Comment:
(bonus) See if you can rework this to use both buttons and React Router Links instead of click events